packages/swap/src/routes/v2/utils.ts

```diff
@@ -176,8 +176,15 @@ export const getDepositInfo = (
       witnessedAt:
         swapRequest?.originType === 'VAULT'
           ? swapRequest.swapRequestedAt.valueOf()
-          : swapRequest?.depositReceivedAt?.valueOf(),
-      witnessedBlockIndex: swapRequest?.depositReceivedBlockIndex ?? undefined,
+          : swapRequest?.depositBoostedAt?.valueOf() ??
+            swapRequest?.depositReceivedAt?.valueOf() ??
```

Comment:
a boosted swap does not have a `depositReceivedAt`

Reply:
probably easier to just check:

```ts
swapRequest?.depositBoostedBlockIndex ??
swapRequest?.depositReceivedBlockIndex ??
swapRequest?.swapRequestedBlockIndex
```